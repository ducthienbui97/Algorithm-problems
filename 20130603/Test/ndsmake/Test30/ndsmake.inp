3051
-389531122 -422830479 -871388052 330792987 680489259 -372164751 -887107312 -638458590 -959906726 186779087 256672672 837524384 620051291 -572687698 444370789 -579359974 -625315217 318417505 -591966545 -110339327 -810778092 207683659 377449082 -669621336 785472879 466688367 -108534988 -486279967 335099364 -819167852 844211100 694897715 568102344 -131628783 -772005946 783430841 396592174 -426034886 41525819 962030234 438094814 -102625945 -159381187 -78349280 -442507420 898109085 -468134361 -643931529 -528669628 683728860 -792751885 441756677 -517498619 -290741802 336769632 527465472 82423397 -330112250 -229378576 -299595462 -314276126 -343651173 -762029789 164981963 -657585472 -389137261 -848915130 581718106 536548625 -598082320 -775494905 -111056561 -631549513 991840764 219508868 385064423 -558845417 207927981 654565135 -483120793 201409116 -667284189 -268787071 -325906068 -794506907 -844186569 -732668319 -222544563 995314604 -630330450 -274519797 -501762412 800001719 -823120914 -171014850 -313128106 722387379 -979050328 232642259 -260686531 310448051 392917442 601096416 236805991 -938672817 -983514343 -376072651 131329692 -818529963 840774922 -681165443 894270497 133207941 -723466795 -854251494 494930443 -176583891 -293787940 746723807 620663677 -452927677 -721847578 -520023793 -991486165 -285582723 -825616836 313367477 -291387512 377446929 508502280 -125039025 -985436558 2051912 589078121 -618220935 -484485232 803878717 -545615193 545715771 395626197 -230804428 850139776 386270699 -69155551 -249976172 963077537 276617920 -958640890 925016962 -984242640 731015157 -829341273 -969214180 -779016314 -35180394 -409685571 -889544625 772333050 843591659 725289139 744830038 697557465 646422142 759502800 930375537 -608649783 73253431 845589372 137422412 569668737 516461343 135744147 958342521 873917637 762287297 520407861 -37714558 -760341054 -396632947 254214668 462770131 685718678 -9683230 979979000 -900766120 98587744 -852362511 -956141314 674264569 -602369823 197311910 408853300 967862307 -693718901 -379823869 965490036 -73584848 149100101 -603555428 317175525 -594608966 -713089681 -355269762 -891806956 276645214 -132786866 -74155669 144926306 261029119 -960753291 -620039337 25115427 -853192701 -709526337 -920950103 846456929 575424580 -747003611 -729670494 -713847556 -780317051 189834422 -731354777 365234923 -343012977 863085475 349526444 951823520 382719212 965433718 184063231 -215974262 -751982273 -516363421 -633195857 -37716300 -556142530 82822583 800485207 878896267 209411662 949001837 203219645 492800953 873778911 -322182116 -458463940 -777279602 -437102950 -876488053 844721505 560938518 -488375990 -562694135 183881262 -779965202 954730964 890279079 -217240916 -304447802 -996174717 107421727 340998259 303204533 736148057 703639486 570189896 560607111 468009196 856504032 822180035 961736441 -222668198 65012748 858805512 628118982 869895244 594343892 -551685293 -103924626 390539881 895401999 -895718961 -450969922 -29695973 845118104 -738952330 -418192090 -693782983 58474096 -395792977 367946583 146989409 687950452 -482974183 984317955 -229077985 321508934 228367730 785154204 146815097 486526298 392005455 -911384322 -678254565 947621622 -223012696 587112789 419610109 816460514 685758545 737259027 453887084 -614443086 157547793 -832697092 291883443 -503557340 848302945 -418707106 -241377946 85444255 324797635 303214943 209360970 -171525668 646773435 562959982 475134180 260210080 -364234898 -58203428 591675037 37796007 -573006378 95516291 775818638 -258269027 -762035296 -811082588 -993606418 -395148987 -786757981 181996751 -418475499 -314584131 93372412 208603387 296118828 -328185896 496324298 221088438 831894644 -908770901 562975875 470290806 246681391 -185983759 -41860083 -907396818 -143906349 599897915 535022072 828289229 -436172298 451103581 -143916396 137524901 951782478 172979012 264937716 -670405733 283536946 308401697 597311486 -434578479 -243992173 837148315 -295670217 930468280 -818473290 278718723 -414694130 -625607542 889994628 -337386849 -865932605 -690864912 640377486 -329381646 -275768893 -161810014 802795838 -950607424 857571055 -149403646 82560847 988892577 169232500 649027655 685671736 -54228029 684735366 -692010542 -694639029 891580186 -424529963 -263768598 960899682 -773058039 681921255 998253687 769361163 634903210 981159840 -473810342 635915743 131021242 -829217821 -839787673 -200973324 -623170012 -885731757 -645891584 792958568 -770885848 -600814029 391876175 -25754594 -873791704 962514795 293701407 -134052731 430811407 635165868 213823064 -865580450 138695082 -101082347 894442194 999799765 874704776 219573092 -811634287 911641274 604935506 -160325735 -959427013 -317557560 821837475 -723054257 -683232572 -506627913 115810300 -117948865 -11852444 848473019 686017840 213208461 346780062 -749680911 -939449750 -574737105 -42006899 326622383 219142807 725157437 -487225666 310543671 -217126616 -726568848 402979790 -902308415 -983557938 -57445543 955397299 413805060 -970613803 212512948 -116097374 409390036 411174938 104550871 -55852384 644010725 307353771 -26832315 -20532823 -211712339 -586338567 -547610272 -584627850 -104436632 393358780 410692922 -395404305 924478261 -569046900 37577517 -366751651 -756057545 305561023 -872051670 90903988 -752315380 966263080 -526286328 914517017 -131582046 471929937 587768827 713305112 -783532643 -236303246 -96145453 807169710 -284972257 -370897409 -322663639 -114020308 -987287717 -874351322 102268724 -289276247 -627551062 -961787157 -807511584 244281234 893777604 354396320 -135529725 415122005 -780458171 -235384548 -652576377 -540705169 545539009 156296750 401521653 -663267948 575531541 141204358 555147796 -5854804 833986520 -879956413 264857256 -912245007 548735770 283092630 -187082430 -513987296 75972794 41936258 -711008460 676399093 662153001 66562571 -936079605 -827469928 672916356 161896665 9686365 314291021 -345565720 -393240935 -82674690 314159678 856454403 -612661767 590058092 521059865 -709149874 378111415 -305710167 118539422 240202927 -616817312 -920931568 -356666662 771814360 404976733 -507508317 791899079 731975294 599967802 -199868574 -218996225 -618052511 586537049 -606778862 325499649 -627057773 -290605222 199510009 877134945 415019708 -234483563 162784310 600427040 -734073337 785515460 -538745158 -38404074 -637515603 -565192514 -249787929 -353939316 -113789668 -534450850 -647442493 447250918 42833347 121069442 729067737 -245414531 -499222403 546379055 -740254036 784064907 -598743854 -461016103 -628340672 -921093248 -35602526 -737047608 694701553 -612376543 222125070 584437872 -470300571 -749330752 665454947 -195116682 -827609438 417993275 -310368707 990503829 -341138311 366725379 913849899 -620353195 808438157 352253945 -655678808 586196709 441343195 3691024 -652121695 496206938 -115496705 -581912636 -582482734 551571544 70518797 -167517349 167443965 -469937405 783639644 -885767946 -349867067 -67572220 399738343 256722098 41421181 974231622 86924728 332634934 -603406002 478117333 -290839261 -921267861 654638049 288749208 520815947 -698610325 551787233 -64169431 894128595 986669170 -422885164 -748626023 255641618 -743197278 -486572003 529688420 -814860798 -815017054 -201842254 -185751880 838908125 187276853 -620677606 324616460 915631470 -744200049 -412741772 417797883 -10086762 -436643988 -81399491 992199138 -396878261 751845740 906824129 -885756699 -639456764 49869078 289392639 147097124 -498423877 -548484689 -649079174 223288739 957346123 394800846 -26358896 -883831201 684954277 31481528 -478623137 104758773 -120984111 -579117110 -654899116 765518978 599659279 450683217 420113887 -21970605 757308914 142132346 634808565 -792708444 11854749 -309360372 679916063 -826003652 -677883345 -463197319 -869556452 182372664 -240347009 462328555 814397467 158839201 251432905 -418277381 -894417663 296615401 -268502305 271465519 -637137372 -277261666 -602491915 -909436169 316110851 -799349546 -343625715 -475618113 342411055 556646526 102396216 -506941389 283561138 -788274667 924617009 -846942436 -709589799 -795267353 -456147744 -439813589 73734234 -313076142 779113690 330546966 -485429814 746834866 847139717 318386038 -37244746 -680450792 89088895 913923116 -717459742 501271607 285435589 -111067490 842358518 -526197351 917275086 661264864 521748891 344597729 374661300 -52291084 392476677 -846393319 -171955842 -63125624 466186584 -694458669 -888940796 -287150602 -120925452 -848534439 -900269454 602753790 -986337711 -998222593 599874641 204745711 -799465378 195221075 252114532 248048707 4966092 451168384 -508918431 984577821 831333719 72348592 186879868 447906946 693702295 517687192 -143669469 582712697 -719865468 -730113635 -217372221 21503022 553139298 670632995 993750908 -428769110 -733240474 162086550 125492939 -821673142 919966001 -784574294 832156 376873409 -710119395 882611433 842435152 251333702 538811265 790088520 652399131 -256224907 877848088 753591126 -725384622 -981046963 -160022923 -547337871 -67441868 808794521 -722421805 347466739 203163344 336835219 -511752431 171003292 -489832556 620942243 -559147698 -621408978 -779627148 -218357219 598479896 23112627 644218120 -702378139 -797655478 936241489 985576626 973508824 974220583 992900625 555529601 -687957746 630396825 -661455322 888740492 133365309 110201150 -702364038 848651592 -909905769 -1957185 -649249894 -332164313 888290368 -315552446 -363510463 682535771 903672565 -431355777 328148665 129345808 -299602229 -976133499 701150136 -521483572 -667581116 -186448368 982841496 653966420 371365988 -363346576 421467551 -800676329 -851089211 -996199628 217958782 327351408 -900737592 -11912749 648653573 -464310553 -241354821 143622600 345856959 -545126362 -646692978 -682145576 -313527450 -344087640 -112994403 -118632059 477733174 -453876763 -63058470 -724201920 -163936868 -511613750 469167426 -568388310 -722938444 786394441 -544164001 -780946360 -768080287 387519445 -698859113 442905501 661829039 469368173 222461415 -407184104 -624396402 -918935438 -921382658 400599506 -189781813 -726800380 766238804 -904074553 -720581482 515325224 158147301 -823462331 -630993391 438807394 328321805 -377258491 -722058188 -634053725 766699702 187856364 -60902449 -704120052 -925293723 -542057944 832809559 327536907 -947996373 -529117865 -896599342 -617536067 936977146 724731582 -18971681 336046768 -655967419 239312354 -987789081 -38081085 966915124 498351413 -441575382 343897985 -511373842 -623957565 403266446 1583602 -844640527 462273410 660328273 280341619 -691558162 -236462653 -187785233 -49834801 -69111418 -122048216 -319253888 70334475 887599015 32310576 403635802 -264011681 89740024 -494018614 262242003 -206313262 737095792 271299450 845135651 -635873167 -176982527 -56652249 331145832 -32950485 -778606734 709503614 -675665468 318420556 407819851 -790532601 -623207775 272361108 838723305 -527204926 89884437 551802355 -71510863 -818007171 -882247072 -35221788 205424481 -597926475 407579538 245592931 547482498 -524889050 -261212086 935534586 -42071310 272599449 147394514 899721918 198274666 -814062332 -289087784 406455972 -326864517 -658814040 -677470481 252705878 885381303 -748820104 315514669 -213624733 -976458753 -732203358 -181232602 794669167 74766539 772443409 929625162 -334689909 -328257770 208638987 -99121189 371085727 -170458392 -61012998 -27124087 -568738300 846716033 823452743 -350368597 858472750 -231718625 -343444964 -909743738 -696238483 70097118 367856816 -967828650 139061690 -176195502 33339358 -579753298 -609413445 570545851 589849476 -521268603 -281489164 870676689 -875021017 164040198 500318312 -952843366 617093124 394875078 -383964453 638086673 541925213 -200751022 982514513 -853749481 635157692 22915813 536039051 -153001954 -855786745 -280327413 -707255710 518160272 -862225300 -624719030 583020055 -315862561 -785160701 -84283006 -976376057 -796131662 -827057120 -844476816 861996813 -253166790 789707065 650880628 -182368860 395102574 -377237613 553474967 374384039 -996051783 244027615 -418856424 -811981764 410247603 811122417 573385657 512494702 -832750127 157285594 673834823 -94019240 523862028 934614671 783090922 -630197197 -254840036 -81426339 469236496 -844274798 110948040 445077859 -629329571 455944766 236294163 714987289 -618958012 -695667759 651226184 -56285239 -38929648 -809041582 -345754014 -16385416 301935851 890313530 -55254406 -819856375 -667193643 488092395 649691839 822826559 527252453 -133471792 -622968446 378590576 -734533202 349241362 -216683922 406748873 -984211011 735980787 859280597 -253427400 -277925570 871350987 -44359387 582421242 749105342 -601272163 716320193 402269706 -758885546 31012172 343513985 -924350303 -179787704 -794581545 -633438789 791303529 -9046979 -564164074 -721685319 -800204308 -53081555 -840453136 -824998566 -797643400 303584899 -871546690 455053910 540924870 -382518172 585903468 753200019 271867376 823374878 -853264124 -456442140 -406853361 -39117122 -295098923 -3370254 -224785291 91586750 377332543 -90556028 -607947302 -447296994 752873257 -952741885 -992299567 476553835 649716473 -552471699 -318020634 -680437690 258941379 186098788 -719556072 -591215971 890373755 -715225932 -83356929 -412392576 -791415229 -896647380 -674654565 357331866 793663801 908928234 -756663547 785689212 -734502317 -781628043 -683559627 -280746861 939580862 379749630 920984663 329849955 547356384 982809353 11019207 -256580730 757618324 669624493 244160241 552326958 -627289243 134483470 -944839140 192616386 -492536792 -661015217 -66812201 121860806 682371893 -367613549 971422498 237510936 -886003879 -969771445 -337325391 883069419 -81773047 588325469 -786408727 -104458205 713233853 154634946 -771802575 -151497793 68967172 -756001174 404965156 318239441 -669003631 -524563807 -256981845 -49373109 -738500276 48307633 -859637985 -870613595 444993610 341460240 -996629973 794093763 -800074810 -539023028 -800632219 -959783211 -339621034 -729219884 -80870663 893295161 509565367 -532654113 605058465 -162704763 -69415464 299244692 -876933142 -168756068 545530920 469651969 789409329 528140041 531680270 948979205 -679931842 -738973956 -549270946 647406558 793964043 995719906 -844869401 125184336 -523610462 -196215649 -159384279 351421550 500931364 501898603 146214948 -222198776 -952424475 -909966010 -796257973 -950508367 -534917707 -19361767 -396661097 120562795 -687006406 270191545 -189536527 -322775431 127913916 677433959 803148961 259296883 -964858750 497972696 -132201027 390052682 -444633603 -255785913 579942848 672429194 24634765 -768040246 -238439796 523518905 -953956740 586947626 892620828 710301953 -606690176 168597692 -300483971 -905196572 -654820280 503047382 911580636 365589336 981921371 -771848747 878834932 801261975 277373137 182328333 -183717935 -493537097 -233441062 -15052290 -685570896 267441895 90577784 -216968171 511782147 -443192248 -737966259 953924379 -16224112 -713999259 -85509746 -635669698 -588283719 111979857 -809603049 -515651142 678659676 242055833 -129144028 -577691535 486092014 259243091 -502349211 -46595569 -208935795 189765611 -539829345 -684523294 -815254759 -704505209 -188119293 -138829058 -781557740 586598275 -770576661 -615835209 475730865 858189923 -76426857 -535244769 -1532730 -315889437 118400369 -414767331 -675191840 -813944421 -159023253 460232998 -769573519 344482639 651672964 627646363 -280203996 716129489 -826239126 671765439 -821146543 -909306477 -857221318 747529555 -276912756 69252813 455613539 -67919664 808449880 151125319 367209128 -827072703 -409341959 806294980 71624516 -670115398 959878247 397116939 -797063641 703330936 -830626722 545911954 -410547873 703194519 -86970989 438468810 185816210 287019882 -609918975 873076226 -470927705 -237603206 -706751136 -222575880 -6722683 -680343778 479829431 -634667005 589674483 -761876812 460325641 -435393066 196436383 322350201 -398256597 -777505039 8283450 -209637877 -251971915 -220287808 168883576 783411223 692805844 556672981 899778306 798074307 -942391031 479643723 716863601 84510856 -935161703 662209729 -403039211 -944984321 -544024561 -656432283 845314846 281114972 -462049575 648832251 -896805143 -260514909 -247883981 -831665419 -342699447 848972192 -900620001 -462644252 354225141 -619311946 -325982944 -655460468 -100274541 -486784781 -985279813 932262597 -825889918 -257245932 169190726 -231720002 -780566707 895715418 -313298845 400423536 -894921012 329632701 -167016174 836546372 -872034290 -848250824 901060091 -192022337 -727690461 -421798329 445629016 -530698875 153195158 -576590820 -664793494 -750732767 -515916503 96873796 54738949 -550069028 -987736041 -935554649 106289488 -626013572 886284455 179246799 168782359 176625977 419089145 -310424798 -644648078 974990691 567721390 834624333 -613552844 59559432 648749132 -272684342 549754309 -249502715 396088923 939174994 -67513534 -28647036 -133780904 771036895 -220117822 392029974 612332994 -315743908 -75700812 -29826967 -863668143 96931564 -679099036 3371686 226603078 96466388 -588155748 -993708439 -326556194 -913159418 -262837401 -69518368 -772514741 865847537 -456109195 -280807594 90309388 -609274943 948260952 -448693828 -192438583 -470861582 -437076477 884989414 -750170675 578065773 -431665271 -512784178 -773675105 66744069 890913890 -68482873 850930611 426246745 -873646687 -203283183 -875752307 990798282 -143719839 -610077076 -761730538 -442379938 570932797 475605139 532817558 492984736 793953884 753691004 419462928 488096849 -997444744 586787130 390502832 302666791 -835883985 -851347508 903367454 126588558 847794369 -665304768 925895766 122810026 488880290 564574249 733924794 -257812680 376232079 -906449107 450054600 -108677305 -296101469 928375457 -150980693 -900327848 386812733 -475037248 -725494546 -939153105 -790523648 139121030 -500361141 -419864263 -805900555 -440605905 931528610 196015563 122746951 -608204027 -205613871 -441175746 108187246 339678960 -864683511 -43967852 -254825302 -303700074 257927423 719024452 -168514455 -243345578 443482508 -610190414 -320972783 -866436532 106079183 -628764139 -690661050 910764292 260450808 -345934470 -592080712 826180682 -114723661 756815073 -235368027 3613725 -354413522 -882638203 -80602382 251249602 -933627959 -294945990 -205731870 -622961420 -238080423 -829160978 -641805712 -295058375 -347397863 -602775217 -776112049 -910142673 735626641 949648252 630699717 -685486122 -305807298 -87729284 -586217561 -891023060 -181622707 -327839150 -864302694 -839907272 -723625615 -184308379 -679372325 -673158078 -708084255 841890776 -912171605 227230425 664803568 637831462 -192915186 471081236 658568719 -155552090 -102617029 -848917157 -381268043 -742462472 72692771 831427109 -278620172 -779379558 -926765663 594049756 659222711 -270904166 505442623 -734249762 100576666 926752358 928984733 -174663322 124339860 -222376218 -600811224 -44184628 606572204 -983466433 -912070932 281606219 -679210596 93727848 931708443 209839074 -538119913 -203809788 405697390 477619092 -309592494 715002510 636415311 160627300 896911635 -607110326 119743906 -526735708 -255879052 458380939 -592627585 -582119500 -834229671 210551150 -518909620 -114718636 267747034 -736895454 631995647 499516816 369922347 514111782 -312886922 198498897 -165434584 -51068175 150661218 -524511758 -178830769 755171943 -546834992 -409032453 734292574 344268115 658708362 663995009 738023376 -149069112 810551897 173390533 707435494 -250620639 194207516 -982068553 -878609672 14923889 570841869 135753566 678773976 -390884624 151695617 -667710341 -63800303 -440802511 -194046007 -459039529 573593031 704687147 -159261258 -877389711 534868740 780548216 -395038873 766551971 -425187619 -428205110 285844733 -441748169 -684940055 -215405860 -69460712 -570371190 -530298232 -760707962 244930012 418680459 768753952 -201722925 -929296892 -248304533 -949960061 5642689 -596318961 -56733983 503775597 -527615852 -204747624 313630487 -663889879 -849143081 -226624317 -17150412 -662339758 -380096528 183074259 -244198265 421107807 571898073 44906829 273525336 125008787 -402677368 -991175283 -306405015 566111011 -97722117 312902 -404231871 -34415977 802755181 324524223 646768549 850981209 964620648 450501659 791183159 583236072 -176374854 -3548972 880496909 -132435255 -58057254 97972527 906398003 131906234 217618332 268508122 -971167611 985337976 935057424 -563036648 -703045543 69410260 521549465 706258127 -633426490 -776239204 -806271600 857072338 -506579633 -939169023 -506008338 630092166 702589684 -314330182 -719011336 -848510688 942477292 88073455 266070346 234743039 -5548837 247331850 537255381 860265923 974459219 640185832 -100189491 -539280127 -628048150 502810684 16785285 -988648346 596990359 319935881 418492614 -678496281 -625830637 796213804 -858806351 -800638174 -283308549 923716349 172805118 -791845104 786814927 896494783 -560819373 891799238 -952602445 -525761077 -659756705 681263192 588975149 -474546595 -433199952 -254594645 -379304820 926092984 631353137 -930019224 -887473930 -770767513 243153988 709895948 -744406515 -283507080 -733886180 605620030 -328290585 -62015370 -30843226 -47325705 -257951697 -813709885 -991100512 565970696 -234014965 966202889 -750862353 347796109 107706205 772701061 -725136662 348392954 11407029 304994030 940574877 -572688830 -78786340 332996700 -192935089 -9638630 27089022 297771855 -383625652 -706532424 -32036128 928833695 714290986 -902586183 -532143951 247750740 -289850743 209994839 36204552 -891680380 -503815092 -475738172 -858085071 247720883 -421730153 412421071 790312283 -129265679 -624720444 637304047 -901776699 -915389087 -445374212 -465508237 -811517053 -852520443 584208346 512793984 -5852816 -697485004 -803892978 921189102 -756342080 -389555250 644762176 -697829209 250107558 944462291 634606867 -272378012 -140750051 -100171482 105868233 336049025 -703935955 -405735321 483050849 -713901372 874935682 962602511 -266792344 875395322 605745346 -567798787 -759012499 283866845 -314872594 -293938125 -793997256 249239136 -12191776 -649730265 -817979366 -636835432 -910394892 -701663801 -544660128 232246855 -188437565 512528653 -495215135 923056133 798123531 -187295430 912132791 585085709 569626026 983912113 734466314 723765665 -112394807 -434647437 -499509753 -780265957 893973014 -865035213 -425859088 -862852206 -673213655 -366775274 787719758 -208756955 53274376 -522109431 -666977471 -141796377 336824262 514989246 -796272648 -678294681 -488332730 -348704188 -595458501 -469959462 -580264163 292934721 -446693083 816026855 64075669 -22050353 971581168 622260239 -230369074 -312095503 -413895191 477888952 -881633536 475933105 301278850 288578004 793940699 -336923433 13866118 -997237966 280023823 696677447 -201963161 986401889 -95308825 -292424968 -567797325 -739095236 823570192 -127411571 -844798043 34331415 107552120 898075794 -93733672 609496453 147797330 924013908 771230998 -652126781 510371382 180646492 311104591 553298628 -968911516 360629898 197157285 469150681 -209658839 -244484661 -668483164 -280222723 -430798246 95109751 46782863 -691669062 -456405251 -192076215 819193301 -250239044 -560363256 508772604 91697648 -825695679 267114110 -976364141 760847193 378369604 -499796269 409607042 946474095 792438691 -513078903 -485276289 420743731 -782768857 -72149292 679439839 568849876 -861552331 -655800251 603604589 436313335 229498188 -569296085 21351614 -635747161 375376811 -128754042 781199779 -233424679 694495504 861615780 -246753863 638093978 479023258 -450363301 -971098854 183518935 124915170 662043702 -851132230 273864524 -933900356 -717905157 -537640915 -546655823 198538745 375682597 283375514 935652772 543634327 -551379382 381340409 577721848 696746971 565034645 -104284797 -917257120 -195549048 -34354676 983803547 229240682 -869656504 450817449 -229961091 28100402 236315252 303532377 -140268745 265962883 -844794080 750525047 -896537782 783122958 551772761 -151755770 -458451844 -345960165 -28358371 120730637 -354672817 234725816 308295795 -426411455 -52334583 -288301473 -691347693 300582737 525772988 -7673631 19371508 -398320581 -796674003 -572284784 642961206 881641886 368470692 -492354492 305388795 14513800 445545811 402911863 -483193888 401883758 -994623903 722717072 -203642089 -51882955 978772881 -245790147 -651281373 -698464147 -535535212 -48120327 342139950 -291368803 513224167 295207853 930470154 -816433735 -385596301 -380755006 190287792 -837291649 -417877387 -233786659 320054306 -958847525 986813428 429708847 -536805841 -996904724 -49873819 -15409573 910946666 580695477 347683304 490691053 823598957 633256877 513280902 826656217 472959857 -873807345 766200873 809576739 15285053 -626031396 137843846 102161829 620726555 -796750003 -958335631 -903426437 -292621373 -522588030 999138444 802211649 473940995 412401338 420972165 -119513963 373762221 855815821 221111631 157691459 -355638895 -408809442 221553797 -39109960 209848732 573686657 984443749 244125567 -7922690 -927459837 85981019 507123985 -970952043 -621961417 843383237 435520062 473725571 -245841620 717576501 -501886522 220355722 -131857334 15666322 692114388 -600833536 -342909345 -711530917 -637378278 -723110177 -665837498 354884108 405421450 -713843975 -478014410 -758635111 -341382995 -733153628 840894743 -237577820 549885027 -513010054 -836288326 558146317 -771853306 134724473 -353433438 -27443224 778149935 745457550 963014993 -223015609 928098679 -10104807 -955826566 -488122847 -947225288 -11307868 220664739 -453435103 844181420 971728193 -321046559 712614617 -960484080 -958489804 -788766441 -978668985 -124872831 -502171312 -446287373 637534694 -64381895 933405456 -138490694 -91362495 262434933 580825096 113556540 -39421064 -134402455 -468511428 404135111 -977593982 464250667 -856473067 143727467 -589675973 -754513768 145932456 -45046442 916731486 -27799420 -242143739 424975183 -555305760 512590092 -310301411 -439682213 965984489 363218952 -740598195 -924728702 311725424 555871133 -908786293 407726251 78727282 281112047 54700332 352843940 928522190 -409470477 -114715825 457336382 -15107648 673483618 212320234 -63314795 164952824 -110599825 566042222 628266070 -383156547 -2898406 415847097 -264474043 585848621 -955512870 -404392139 -400406607 784297936 640979161 -555637965 -258394 571027645 -99017046 -226286478 -18220922 -131347431 -525758962 -745947095 802220007 16514026 160061795 58769956 21917191 701926154 785273770 857119750 -151722516 -781399501 392165037 986703644 -482166199 -489695513 26687410 465234422 -635996036 -494318286 -440951537 764909484 -126594582 -109491317 -404541220 160358366 -63628004 687142706 98536915 -154817757 202692656 804173654 -913698222 177964844 -541530084 -470155477 -213639064 65201226 697386285 767895913 27188522 -867551690 -629342715 426852843 -600591903 387264672 -46773250 951480793 750114190 -910715218 -786808420 -612558250 -207209887 705228844 149320767 79549348 5650959 -516598611 -516557939 940349151 -355039203 975072559 15803999 765112777 62205953 -577898561 541319870 -291582134 407893197 -647144737 -971069818 -49821905 295232100 -172128665 328692348 -996071964 -592337776 912263862 13456515 613002068 963749004 707101738 -475229393 -552611672 105859235 -96859222 -791296025 139569197 -571518105 -449388960 -756797002 368622730 153194116 -813028690 -459826479 -468255396 43265876 965127398 142240481 -229700064 -977766981 -733620338 481012353 -785777109 122225125 131225248 284278698 -922665499 172673798 606444314 210391254 911889930 -369932653 -905638823 614058721 573055903 618815151 -557224717 -242244069 278222646 715619190 -786871721 -376606257 -245645097 -130458096 462903145 263011417 970110845 585962414 866709502 -238134570 519865086 316540003 -537677564 -430028836 293331122 -815215062 -509751066 848039130 -447008506 -224364448 -726815897 68598782 942464187 107066316 -111334134 -191076221 252350251 -680897527 -742799702 646366068 -507059329 297681650 -809963370 289149830 552109237 270743818 392939036 -11918448 535729825 235545000 125784657 702175646 -705674049 -649393262 172452756 978478822 -922754098 -238241598 752354164 -812465121 900718186 -405850271 670601366 451144663 427195265 -115093165 23422061 911513301 -490802135 854972385 -554595758 -269609315 -241487421 -840393209 -680367559 -121429231 805248558 534130749 -761248717 -28099377 -496607220 287977309 -584343027 -115631322 -3126721 90124331 153944342 -914440267 753799284 -478262906 551949822 123955618 -154242347 -242148566 -157274918 201146996 764621538 -932104416 644433732 465387150 -219167910 -521898045 497662624 772623177 -277494445 -433820562 52834425 -656767828 472581225 627549895 -65139959 -257241910 -458374125 409359011 112082617 -865644527 784343701 558979707 982161194 -65412607 -834651370 -250758787 891702442 597036562 -52664750 144357212 -837111436 917833935 731347597 -282906198 173021424 -134727651 331578672 -228097964 490511052 -880565952 522103924 -427733248 -420631114 -240130537 -946914902 -93820273 -250711117 -632708248 125050404 -580934462 136327711 64954966 742266051 293670904 834146114 576351983 552231455 -460131051 476893066 481625570 -576080564 -834468481 737373006 -937948337 805006146 -436142384 -789963595 814685981 920484180 781457212 -810389489 602640460 -567812622 -71615102 -162721285 775260761 998922537 263647773 148436240 77616789 -305718928 102574893 337683538 14738092 279052402 -950540529 70060736 865199571 966799850 -185856361 -953425397 803008142 -207720792 -429319583 311051035 -773136976 501531393 847097190 915283582 645334401 -987599700 -766808508 738252659 -58192540 -721123111 580191305 282971962 -112401764 -948534769 -23319896 -410957689 -425436746 524004170 -105811390 -16298803 948696998 141761274 -485255321 -769705183 -119414748 -347186097 816988809 -507130822 -167094643 -817551348 836265379 -512486248 87201909 -111229369 121757999 891578978 200859202 206509991 -859557711 501669101 -592132314 -754326167 -654691565 239953807 600309376 84325242 937178138 -974973153 -265227760 167367814 675569781 -760534092 684620186 -419057952 -363071403 -990076771 236731806 14357415 -403636699 -444339349 924066040 -26389286 -783235857 -802980152 -884319054 -164629316 848303035 -930988718 911263272 521702433 455832201 -904926596 135100554 -372877510 877830216 759416500 743156097 825039617 -127088320 749876752 -872714623 305353771 259270011 -46644930 -885126212 -630970266 -932632636 388657338 259124447 -472172022 -980074046 257060973 -718327823 456672953 -947855142 345646330 265187876 -933482403 -485051076 -503556674 923024546 -209966466 -201236292 678378655 -546013176 518881006 -563475652 -706092690 -680735119 440469782 -537546896 -545908380 465419090 301412336 313264844 -286414667 -777311869 -126159752 -853822570 -119604774 99924985 -339021471 -358615569 -351971362 696694366 224900969 -538604641 166174713 -923141385 607678910 -745426074 -428013631 -107440835 904837192 678139508 330214604 385480053 504384572 -24854901 -646733677 687410157 -105985074 536511699 451842163 555654612 450998019 -224173254 -300504116 -811217482 181010340 436187781 925731404 40268180 682384974 -562823432 -44174266 -355438436 530908839 -193228996 -71129941 -706360266 -669413252 135154331 -837817052 584631738 -433145883 610494505 -765392072 -462170739 626980043 -886130968 -524741348 -295611584 -924371216 -409102149 -403908121 16453888 808324531 102548276 -85230975 95741600 843306821 -247879001 -353738822 662741823 4678165 -38191241 391589782 -791674290 979762602 -639892729 -943371552 318396012 80969875 -644420506 651447863 -655668193 542841268 -702352474 -459457445 -126848737 -236359393 -153971664 788505657 -958690949 517042620 984663019 110605007 482393939 -319301925 -929657176 -460895930 78327847 38757624 -449263248 -981097831 915751069 -51719429 31719315 580418795 81662933 -62110694 -359426219 489700997 -805968422 -131975572 99311269 -273207188 871914292 -988770292 257810952 910919835 -561317990 -886762266 718147477 81632738 -179892796 77601 539019289 -739819685 148478559 345710682 17222020 737713178 968111225 479736574 108305118 128148288
