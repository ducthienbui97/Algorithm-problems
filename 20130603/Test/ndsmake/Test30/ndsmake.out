1527573292149
